spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
```

Comment:
Do you foresee having separate helps for each of the models effectively? If so, will there be some overlap in terms of having `EquipmentModelHelper#mock_category` and `CategoryHelper#mock_category`? I'm just trying to picture what the final `spec/support` directory looks like and what we're including where.